contrib/runners/mistral_v2/mistral_v2.py

```diff
@@ -69,6 +59,15 @@ def pre_run(self):
             self._notify = getattr(self.liveaction, 'notify', None)
         self._skip_notify_tasks = self.runner_parameters.get('skip_notify', [])
 
+    def _get_client(self, ctx=None):
+        auth_token = (
+            ctx['auth_token']
+            if ctx and 'auth_token' in ctx and ctx.get('auth_token')
+            else getattr(self.auth_token, 'token', None)
+        )
+
+        return utils.get_client(self._base_url, auth_token=auth_token)
```

Comment:
So if an auth token is in the context, this will use auth_token over API key? When is a legit case when we would use token vs a key? Is it when API key is not supplied in config?